src/material/chips/_m3-chip.scss

```diff
@@ -49,8 +49,11 @@
       chip-selected-label-text-color: map.get($system, on-secondary-container),
       chip-selected-trailing-action-state-layer-color: map.get($system, on-secondary-container),
       chip-selected-trailing-icon-color: map.get($system, on-secondary-container),
-      chip-trailing-action-focus-state-layer-opacity: map.get($system, focus-state-layer-opacity),
-      chip-trailing-action-hover-state-layer-opacity: map.get($system, hover-state-layer-opacity),
+      // Parent chips will show focus/hover state as well so need stronger focus/hover styling here
+      chip-trailing-action-focus-state-layer-opacity:
+        calc(map.get($system, focus-state-layer-opacity) * 2),
```

Comment:
Either we should have a specific value for these, or we should calculate them using Sass at compile time, rather than at runtime (IIRC the values in `$system` are static).